### ISSUE TITLE
Cannot read property 'dependencyLib' of undefined

### ISSUE THREAD

**Issue by yogi12314** [RobinHerbots/Inputmask#2369]
Error on submit form.
`Uncaught TypeError: Cannot read property 'dependencyLib' of undefined
    at HTMLInputElement.ev (jquery.inputmask.js?1596795273:1451)
    at HTMLFormElement.dispatch (jquery-3.1.1.min.js?20191214:3)
    at HTMLFormElement.q.handle (jquery-3.1.1.min.js?20191214:3)
    at Object.trigger (jquery-3.1.1.min.js?20191214:4)
    at HTMLFormElement.<anonymous> (jquery-3.1.1.min.js?20191214:4)
    at Function.each (jquery-3.1.1.min.js?20191214:2)
    at r.fn.init.each (jquery-3.1.1.min.js?20191214:2)
    at r.fn.init.trigger (jquery-3.1.1.min.js?20191214:4)
    at r.fn.init.r.fn.<computed> [as submit] (jquery-3.1.1.min.js?20191214:4)
    at order.js?1596793641:3384`

Windows 10 
Edge Browser

I used jquery.inputmask.js, lastest version

**Comment by RobinHerbots** [comment]
@yogi12314 , 

Can you create a codepen or jsfiddle for it.  I cannot reproduce here.

**Comment by yogi12314** [comment]
https://codepen.io/yogi12314/pen/QWNbxjR?editors=1010

Here's a little part of my code. Hope that help you

**Comment by yogi12314** [comment]
Ok i finded solution. When i have defined inputmask for input by class selector frist, and  in next step add next inputmask by id, then problem occurs.

**Comment by payapula** [comment]
@RobinHerbots  I am getting the same error but on a completely different use case, and I was able to reproduce in codesandbox [React] - https://codesandbox.io/s/inputmask-undefined-error-yqug4

In order to get the error you need to 
1. Click on button 'Enable the Mask'
2. Then Click on button 'Remove the Mask'
3. Enter any value (you can even leave it blank)
4. Click on Submit.

You will be getting this error - _Uncaught TypeError: Cannot read property 'dependencyLib' of undefined_

Upon investigation, I found that without enabling the Mask, the Input element don't have the property **inputmask**. However after enabling the mask we have new property for the input with key **inpuptmask**

After clicking on Remove mask this property is set to undefined. **inputmask: undefined**

Which is getting evaluated on the code as undefined when we are submitting the form.

The below piece of code is producing error and its from **_node_modules/inputmask/dist/inputmask.js_**

```
   var EventRuler = {
            on: function on(input, eventName, eventHandler) { 
                 var $ = input.inputmask.dependencyLib, ev = function ev(e) {
                    e.originalEvent && (e = e.originalEvent || e, arguments[0] = e);
                    var that = this, args, inputmask = that.inputmask, opts = inputmask ? inputmask.opts : void 0, 
   {/*error line*/}  $ = inputmask.dependencyLib;
                 <...>
             }
 } }
```

I am not sure but undefined check on the **inputmask** could resolve this issue hopefully.

![image](https://user-images.githubusercontent.com/7134153/94701789-ae3b6480-035a-11eb-9b3f-3874b355bbd8.png)

Below is the screenshot of error in dev tools.
![image](https://user-images.githubusercontent.com/7134153/94701987-e2af2080-035a-11eb-9ee9-9bcdaf349a3b.png)


Please let me know if you need any additional details.

**Comment by RobinHerbots** [comment]
@yogi12314 , 

The issue should be fixed in version 5.0.6-beta.19.  Also see #2403  

Best regards, 
Robin
